sale_order_type/reports/sale_report.py

```diff
@@ -0,0 +1,19 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class SaleReport(models.Model):
+    _inherit = "sale.report"
+
+    type_id = fields.Many2one(
```

Comment:
may we change field name from `type_id` to `sale_type_id`? Same as you declared in Invoice analysis report.

Reply:
IMO it's ok like this as it's the same name in the sale.order model 🤷‍♂️

Reply:
Ah. Got it. :+1: